src/router.rs

```diff
@@ -11,13 +12,20 @@ use iron::Url;
 use recognizer::Router as Recognizer;
 use recognizer::{Match, Params};
 
+
+pub struct RouterInner {
```

Comment:
It's not actually a public type, since the `router` module is not public and this type is not reexported.